src/__tests__/mapAndApiStateNames.test.js

```diff
@@ -0,0 +1,25 @@
+import {
+  getStatesAndDistrictsFromAPI,
+  getStatesAndDistrictsFromMaps,
+} from '../testUtils/index';
+
+describe('Compare the map and the API states and districts', () => {
+  test('for any discrepancies', async () => {
+    const statesAndDistrictsFromAPI = await getStatesAndDistrictsFromAPI();
+    const statesAndDistrictsFromMaps = await getStatesAndDistrictsFromMaps();
+
+    const statesFromAPI = Object.keys(statesAndDistrictsFromAPI);
+    const statesFromMaps = Object.keys(statesAndDistrictsFromMaps);
+
+    expect(statesFromAPI).toContain(...statesFromMaps);
+
+    statesFromAPI.forEach((state) => {
+      const expectedDistricts = statesAndDistrictsFromMaps[state]?.sort();
+      const receivedDistricts = statesAndDistrictsFromAPI[state]?.sort();
+
+      if (expectedDistricts !== undefined && receivedDistricts.length !== 0) {
+        expect(expectedDistricts).toContain(...receivedDistricts);
```

Comment:
The data from the API should be available within the data collected from the `public/maps/*` folder.
The maps data can have more states then the API data because we may not have active cases in all the states.